packages/uniforms-material/src/ErrorField.tsx

```diff
@@ -1,3 +1,4 @@
+import { PropTypes, useTheme } from '@material-ui/core';
```

Comment:
Can you check whether it's emitted in the JavaScript code? Again, I'd rather not import the entire `@material-ui/core` anywhere.